### PR TITLE
Change setUserProperty:toValue: to accept an `id` value.

### PR DESCRIPTION
**Pull request by alloy**

### CODE REVIEW

File: Providers/IntercomProvider.m [hunk 1]
```diff
@@ -33,7 +33,7 @@ - (void)identifyUserWithID:(NSString *)userID andEmailAddress:(NSString *)email
     }
 }
 
-- (void)setUserProperty:(NSString *)property toValue:(NSString *)value
+- (void)setUserProperty:(NSString *)property toValue:(id)value {
```
Comment (orta) [inline Providers/IntercomProvider.m]: this has an extra {

Comment (alloy) [inline Providers/IntercomProvider.m]: Hah, and it’s the _one_ that I could have catched by running the tests locally :smile:

**Comment by alloy** [comment]
Fixed.

**Comment by orta** [comment]
:+1:

**Pull request merged**